scripts/go/sherpa_onnx.go

```diff
@@ -593,7 +598,20 @@ func (s *OfflineStream) GetResult() *OfflineRecognizerResult {
 	defer C.SherpaOnnxDestroyOfflineRecognizerResult(p)
 	result := &OfflineRecognizerResult{}
 	result.Text = C.GoString(p.text)
-
+	result.Lang = C.GoString(p.lang)
+	result.Emotion = C.GoString(p.emotion)
+	result.Event = C.GoString(p.event)
+	n := int(p.count)
+	result.Tokens = make([]string, n)
+	tokens := (*[1 << 28]*C.char)(unsafe.Pointer(p.tokens_arr))[:n:n]
```

Comment:
By the way, have you tested the changes?

Could you update the  examples to output the newly added fields?

Also, please consider the case where p.timestamps may be a nullptr.

Reply:
Should it be

**C.char

Reply:
`(*[1 << 28]*C.char)` is correct based on what i have observed in code elsewhere

Reply:
> By the way, have you tested the changes?
> 
> Could you update the examples to output the newly added fields?
> 
> Also, please consider the case where p.timestamps may be a nullptr.

added check for null timestamps

Reply:
> `(*[1 << 28]*C.char)` is correct based on what i have observed in code elsewhere 
> 

Please test both methods and see which one is correct.